specification/dimensions/tags.md

```diff
@@ -0,0 +1,76 @@
+# Tags
+
+The Tags column represents the set of finalized user-defined and/or provider-defined tags assigned to Tag Sources.  Tags are commonly used for scenarios like adding business context to billing data to identify and accurately allocate charges.
```

Comment:
Option 1:
The Tags column represents the set of tags assigned to Tag Sources with respect to potential provider-defined or user-defined tag evaluations.

Option 2:
The Tags column represents the set of tags assigned to Tag Sources that respect potential provider-defined or user-defined tag evaluations.

Option 3:
The Tags column represents the set of tags assigned to Tag Sources that also account for potential provider-defined or user-defined tag evaluations

Reply:
@flanakin @udam-f2 @mike-finopsorg Thoughts

Reply:
I think any of those options are an improvement.  The first time we see the term "finalized" then follows in the second paragraph, and we are immediately told what it means.

Reply:
I am not educated enough, so "finalized" and "tag sources" look mysterious too me in first sentence, and I believe I understand actually the first sentence  when reading the following sentences.

Here is a proposition. Feel free to ignore if irrelevant

> The Tags column represents key-value pairs that are either user-defined or provided-defined and that result from an evaluation process, knowing that original tags can be defined at resource level or at different levels of a resource group hierarchy (if supported by the Provider).

Reply:
Going to go with Option 3